docs/runtimes/create-and-use-functions.ipynb

```diff
@@ -431,13 +431,11 @@
    "metadata": {},
    "source": [
     "```python\n",
-    "fn = mlrun.new_function(\"serving\", kind=\"serving\", image=\"mlrun/mlrun\")\n",
+    "fn = mlrun.project.set_function(name=\"serving\", kind=\"serving\", image=\"mlrun/mlrun\", with_repo=True)\n",
```

Comment:
Pretty sure here too, `mlrun.project.set_function` shouldn't be called as if it were a static method.